packages/ckeditor5-utils/src/focustracker.ts

```diff
@@ -155,3 +161,8 @@ export default class FocusTracker {
 
 mix( FocusTracker, DomEmitterMixin );
 mix( FocusTracker, ObservableMixin );
+
+type ObservableDomEmitter = DomEmitter & Observable;
```

Comment:
Workaround. `interface FocusTracker extends DomEmitter, Observable {}` would complain about `listenTo` not being the same.